roles/reproducer/tasks/configure_controller.yml

```diff
@@ -291,10 +291,12 @@
             }}
           cifmw_architecture_automation_file: >-
             {{
-              (ansible_user_dir,
-               'src/github.com/openstack-k8s-operators',
-               'architecture/automation/vars/default.yaml') |
-               path_join
+              (
+                ansible_user_dir,
+                'src/github.com/openstack-k8s-operators',
+                'architecture/automation/vars'
+                cifmw_arch_automation_file | default('default.yaml')
```

Comment:
care to add that new top-level parameter in the docs/sources/usage/01_usage.md ?